components/Navbar.vue

```diff
@@ -376,7 +375,7 @@ export default class NavbarMenu extends mixins(
     window.addEventListener('scroll', this.onScroll)
     document.body.style.overflowY = 'initial'
     window.addEventListener('resize', () => {
-      this.isMobile = window.innerWidth < 1024 ? true : isMobileDevice
+      this.isMobile = window.innerWidth < 1024
     })
```

Comment:
I don't see the removeEventListener below. could you pls add it?

Reply:
added, I'd like to know if we need to remove the event listener for resizing if the resizing can happen anytime while using the app. (i.e., should we be listening to it all the time?) Navbar is part of the layout everywhere.

Reply:
Yes, we should listen `resize` all the time. the resize event only be triggered when we change the window size.